packages/redux/src/users/returns/reducer.ts

```diff
@@ -0,0 +1,60 @@
+import * as actionTypes from './actionTypes';
+import { AnyAction, combineReducers, Reducer } from 'redux';
+import type * as T from './types';
+
+export const INITIAL_STATE: T.UserReturnsState = {
+  result: null,
+  error: null,
+  isLoading: false,
+};
+
+const isLoading = (
+  state = INITIAL_STATE.isLoading,
+  action: AnyAction,
+): T.UserReturnsState['isLoading'] => {
+  switch (action.type) {
+    case actionTypes.FETCH_USER_RETURNS_REQUEST:
+      return true;
+    case actionTypes.FETCH_USER_RETURNS_FAILURE:
+    case actionTypes.FETCH_USER_RETURNS_SUCCESS:
+      return INITIAL_STATE.isLoading;
+    default:
+      return state;
+  }
+};
+
+type ErrorAction = T.FetchUserReturnsFailureAction;
+
+const error = (
+  state = INITIAL_STATE.error,
+  action: AnyAction,
+): T.UserReturnsState['error'] => {
+  switch (action.type) {
+    case actionTypes.FETCH_USER_RETURNS_REQUEST:
+      return INITIAL_STATE.error;
+    case actionTypes.FETCH_USER_RETURNS_FAILURE:
+      return (action as ErrorAction).payload.error;
+    default:
+      return state;
+  }
+};
+
+const result = (
+  state = INITIAL_STATE.result,
+  action: AnyAction,
+): T.UserReturnsState['result'] => {
+  switch (action.type) {
+    case actionTypes.FETCH_USER_RETURNS_SUCCESS:
+      return (action as T.FetchUserReturnsSuccessAction).payload;
+    default:
+      return state;
+  }
+};
+
+const userReturnsReducer: Reducer<T.UserReturnsState> = combineReducers({
+  isLoading,
+  result,
+  error,
+});
+
+export default userReturnsReducer;
```

Comment:
In my point of view, I think we can reuse the **createReducerWithResult()** function. This function already does what you did here. 

```suggestion
const userReturnsReducer = createReducerWithResult(
  'FETCH_USER_RETURNS',
  INITIAL_STATE,
  actionTypes,
);

export default userReturnsReducer;
```

Reply:
done 👍🏻